frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -29,8 +53,8 @@ async function pollFunnel(params = {}) {
     return result
 }
 
-const cleanFunnelParams = (filters) => {
-    return {
+const cleanFunnelParams = (filters: Partial<FilterType>): Partial<FilterType> => {
+    const params = {
```

Comment:
No need to declare params here, I think. You can use `=> ({ })` to return the object with less verbose syntax.